client/my-sites/sharing/buttons/buttons.jsx

```diff
@@ -124,7 +124,6 @@ module.exports = protectForm( React.createClass( {
 					initialized={ this.isInitialized( [ 'settings', 'buttons' ] ) }
 					saving={ this.state.isSaving } />
 				<ButtonsOptions
-					site={ this.props.site }
 					postTypes={ this.props.postTypes.get( this.props.site.ID ) }
```

Comment:
Yes, and it's already the case in https://github.com/Automattic/wp-calypso/pull/10278 I split this refactoring into three separate PRs to avoid big ones